SimplenoteUITests/EmailLogin.swift

```diff
@@ -14,7 +14,18 @@ class EmailLogin {
         backButton.tap()
     }
 
-    class func logIn(email: String, password: String) {
+    class func logIn() {
+        var testAccount = ProcessInfo.processInfo.environment["UI_TEST_ACCOUNT"]!
+
+        // Use 'default' account if test account was not passed via environment variable
+        if testAccount.isEmpty {
+            testAccount = testDataEmail
+        }
+
+        EmailLogin.logIn(testAccount, testDataPassword)
```

Comment:
To minimize the changes not directly related to the main logic that will come next, the tests currently assume that all test credentials share the same password. Because of this, we only have to pass the email at the moment.